app/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "playbook",
+  "name": "Agile-Solicitation-Builder",
```

Comment:
The package.json spec says names have to be all lowercase.  Even though we'll probably never be publishing this to npm and it won't ever come up, should we still conform to the spec?